src/BrowserKitDriver.php

```diff
@@ -236,8 +236,14 @@ public function setCookie($name, $value = null)
             return;
         }
 
+        try {
+            $domain = parse_url($this->getCurrentUrl(), PHP_URL_HOST) ?: '';
+        } catch (DriverException $exception) {
```

Comment:
When you'll get `DriverException`? If that is when session isn't started, then better to explicitly check for that.